src/rebel.re

```diff
@@ -0,0 +1,497 @@
+/*
+ * vim: set ft=rust:
+ * vim: set ft=reason:
+ */
+
+open Core.Std;
+open Yojson.Basic;
+open Jenga_lib;
+
+/*
+  =====================================================
+    COPIED CODE BEGIN
+  =====================================================
```

Comment:
All of the Coped code can be removed when Reason supports 4.03 and npm-opam/jenga use 114 major